plugins/field-grid-dropdown/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "blockly-field-grid-dropdown",
+  "private": true,
+  "version": "0.20201007.0",
+  "description": "A Blockly dropdown field with grid layout.",
+  "scripts": {
+    "build": "blockly-scripts build",
+    "clean": "blockly-scripts clean",
+    "lint": "blockly-scripts lint",
+    "prepublishOnly": "npm run clean && npm run build",
+    "start": "blockly-scripts start",
+    "test": "blockly-scripts test"
+  },
+  "main": "./dist/index.js",
+  "module": "./src/index.js",
+  "unpkg": "./dist/index.js",
+  "author": "",
+  "keywords": [
+    "blockly",
+    "blockly-plugin",
+    "blockly-field",
+    "grid-dropdown"
+  ],
+  "homepage": "https://github.com/moniika/blockly-samples/tree/master/plugins/field-grid-dropdown#readme",
```

Comment:
Update homepage, repository:url, and bugs to point to google/blockly-samples instead of your fork.

Reply:
Fixed